@ndlib/gatsby-theme-marble/src/components/Shared/NDBrand/Breadcrumbs/index.js

```diff
@@ -13,21 +34,37 @@ export const NDBrandBreadcrumbs = ({ variant, breadcrumbs, currentPageTitle, ...
     listStyle: 'none',
     fontSize: 0,
     '& li': {
-      display: 'inline-block',
+      display: ['none', 'none', 'none', 'inline-block'],
       mr: '0.75rem',
+      '&:nth-last-child(2)': {
+        display: 'inline-block',
+      },
     },
+
   }
   return (
     <ol varaint={'breadcrumbs.' + variant} sx={sx} {...props}>
       <li><Link variant='breadcrumb' to='/'>Home</Link> › </li>
       {breadcrumbs.map((item) => {
-        return (<li key={item.url}><Link variant='breadcrumb' to={item.url}>{item.title}</Link>›</li>)
+        return (<li key={item.url}><Link variant='breadcrumb' to={item.url}>{trimTitle(item.title)}</Link>›</li>)
       })}
-      <li>{currentPageTitle} › </li>
+      <li>{trimTitle(currentPageTitle)} › </li>
     </ol>
   )
 }
 
+const trimTitle = (title) => {
+  if (title.length > 60) {
+    const splitTitle = title.split(/[:_([]/)
+    if (splitTitle[0].length > 60) {
+      // match the first 60 characters up to a space or end of line
+      return splitTitle[0].match(/.{1,60}(\s|$)/g)[0].trim() + '...'
```

Comment:
This won't work right if there are more than 60 characters before the first space. (Will actually get the last 60 characters before the space.) But I guess that's an obscure enough scenario that it probably won't come into play? :)

Also, adding the ellipsis would change a 61 character string so it is now 63 characters. Maybe match the first 57 chars instead if you want a hard limit of 60.

Reply:
in general i am not considering 60 to be a very hard limit.  if it gets truncated to close to 60 I am fine with the outcome.  

As far as the first thing.  It is in part why I chose 60 and not 50 to try to minimize that case but I am going to change the comment to reflect this issue